documentation/documents/psmoduledevelopment/templates.md

```diff
@@ -42,6 +42,7 @@ Together the two cover the two basic steps to using the template system.
 Since this component has some complexities that need to be dealt with, some in-depth tutorials have been provided:
 
 ### Working with templates
+ - [How to create a new PSFProject tempate for a new PowerShell project](templates/template-psfproject-howto.html)
```

Comment:
A how-to on a default template is a fundamentally different kind of document to the ones explaining how to work with the system. So how about moving this to a dedicated header _above_ "Working with templates" named "Default Template Guides"

Also: The text on the link could be do with some rewording:

How about "PSFProject: Setting up a PowerShell Module project with CI/CD in no time"?

Reply:
Worked with Fred to update changes for master branch